standards/security_standards.md

```diff
@@ -0,0 +1,440 @@
+
+4.1. Access Control - is the process that limits and controls access to resources of a computer system.
+
+4.2. OWASP - The Open Web Application Security Project (OWASP) is a worldwide free and open community focused on improving the security of application software.
+
+4.3. Input Validation - is a programming technique that ensures only properly formatted data may enter a software system component.
+
+4.4. Encoding - (commonly called “Output Encoding”) involves translating special characters into some different but equivalent form that is no longer dangerous in the target interpreter, for example translating the < character into the &lt; string when writing to an HTML page.
+
+4.5. HTML sanitization - is the process of examining an HTML document and producing a new HTML document that preserves only whatever tags are designated "safe" and desired.
+
+4.6. Authorization - is the concept of allowing access to resources only to those permitted to use them.
+
+4.7. Malicious Code – Code introduced into an application during its development unbeknownst to the application owner, which circumvents the application's intended security policy. Not the same as malware such as a virus or worm.
+
+4.8. Allow list – A list of permitted data or operations, for example a list of characters that are allowed to perform input validation.
+
+4.9. Authentication – The verification of the claimed identity of an application user.
+
+4.10. Cross-Site Scripting (XSS) – A security vulnerability typically found in web applications allowing the injection of client-side scripts into content.
+
+4.11. Globally Unique Identifier (GUID) – a unique reference number used as an identifier in software.
+
+4.12. Personally Identifiable Information (PII) - is information that can be used on its own or with other information to identify, contact, or locate a single person, or to identify an individual in context.
+
+4.13. Transport Layer Security (TLS) – Cryptographic protocols that provide communication security over a network connection.
+
+4.14. What You See Is What You Get (WYSIWYG) - A type of rich content editor which shows how the content will actually look when rendered rather than showing the coding used to govern the rendering.
+
+4.15. Vulnerability - A vulnerability is a hole or a weakness in the application, which can be a design flaw or an implementation bug, that allows an attacker to cause harm to the stakeholders of an application.
+
+4.16. Stakeholders - Stakeholders include the application owner, application users, and other entities that rely on the application.
+
+4.17. Phone home - phoning home is a term often used to refer to the behavior of security systems that report network location, username, or other such data to another computer.
+
+5\. **DOCUMENT HISTORY**
+
+| **Editor** | **Details of Change** | **Effectivity Date** |
+| --- | --- | --- |
+|     |     |     |
+| --- | --- | --- |
+|     |     |     |
+| --- | --- | --- |
+```
```

Comment:
We don't need this @arcubalino - we have git history for this

Reply:
Removed